cmd/orb.go

```diff
@@ -50,6 +53,29 @@ func newOrbCommand() *cobra.Command {
 	return orbCommand
 }
 
+type orb struct {
+	Commands  map[string]struct{}
+	Jobs      map[string]struct{}
+	Executors map[string]struct{}
+}
+
+func addOrbElementsToBuffer(buf *bytes.Buffer, name string, elems map[string]struct{}) {
+	if len(elems) > 0 {
+		buf.WriteString(fmt.Sprintf("  %s:\n", name))
+		for key := range elems {
+			buf.WriteString(fmt.Sprintf("    - %s\n", key))
+		}
+	}
+}
+
+func (orb orb) String() string {
```

Comment:
I know that `String()` is a [convention of golang](https://golang.org/pkg/fmt/#Stringer), but I'm still not sure I feel about it.

Probably not worth fussing over, but generally find this PR pleasing.

How do you feel about these formatting functions outside of the `cmd` namespace?